backend/services/implementations/__tests__/testSessionService.test.ts

```diff
@@ -25,10 +29,14 @@ describe("mongo testSessionService", (): void => {
   it("createTestSession", async () => {
     const res = await testSessionService.createTestSession(mockTestSession);
 
-    expect(res.id).not.toBeNull();
-    expect(res).toMatchObject({
-      id: res.id,
-      ...mockTestSession,
-    });
+    // TODO: uncomment when results are added to test session response object
+    // assertResponseMatchesExpected(mockTestSession, res);
```

Comment:
There's missing logic in `createTestSession` that doesn't map the Results when returning the response object. Should create a separate ticket for it.

Reply:
i omitted the `Results` field because it will be empty when a test session is first created, and was thinking we could add that attribute when a test session is updated. or do you think it would be better to have an empty `Results` field? I was having some trouble figuring out the typing for that

Reply:
ah i see - that's fine! i changed the test up to reflect this